templates/saas.amazonaws.com.yaml

```diff
@@ -83,8 +127,12 @@ identities:
     name: sns
     dns_pattern: .*sns.*.amazonaws.com
     dst_port: 443
-  pushdown:
-  - dns_pattern
```

Comment:
same here. Do we need to add pushdowns for all of these?

Reply:
i was thinking that, what are your thoughts